src/macaron/__main__.py

```diff
@@ -197,14 +197,31 @@ def verify_policy(verify_policy_args: argparse.Namespace) -> int:
         show_prelude(verify_policy_args.database)
         return os.EX_OK
 
+    policy_content = None
     if verify_policy_args.file:
         if not os.path.isfile(verify_policy_args.file):
             logger.critical('The policy file "%s" does not exist.', verify_policy_args.file)
             return os.EX_OSFILE
 
         with open(verify_policy_args.file, encoding="utf-8") as file:
             policy_content = file.read()
+    elif verify_policy_args.policy:
+        policy_dir = os.path.join(macaron.MACARON_PATH, "resources/policies/datalog")
+        available_policies = [policy[:-12] for policy in os.listdir(policy_dir) if policy.endswith(".dl.template")]
+        if verify_policy_args.policy not in available_policies:
+            logger.error(
+                "The policy %s is not available. Available policies are: %s",
+                verify_policy_args.policy,
+                available_policies,
+            )
+            return os.EX_USAGE
+        policy_path = os.path.join(policy_dir, f"{verify_policy_args.policy}.dl.template")
+        with open(policy_path, encoding="utf-8") as file:
+            policy_content = file.read()
+        if verify_policy_args.package_url:
```

Comment:
Let's check that the PURL using the `packageurl` library, which is already a dependency of Macaron:
```
try:
    PackageURL.from_string(verify_policy_args.package_url)
except ValueError as error:
    logger.error("The package url %s is not valid.", verify_policy_args.package_url)
    return os.EX_USAGE